include/seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp

```diff
@@ -386,7 +386,7 @@ typename range_type::iterator insert_gap(range_type & rng,
 
 /*!\brief An implementation of seqan3::AlignedSequence::erase_gap for ranges with the corresponding
  *        member function erase_gap(it).
- * \ingroup seqan3::AlignedSequence
+ * \ingroup aligned_sequence
```

Comment:
why this change? I am pretty sure this is the correct submodule to group it to.

Reply:
http://docs.seqan.de/seqan/3-master-user/group__aligned__sequence.html#ga0c6fc7bb7f0554a0085e521b214c812c
Look at erase_gap 3/4, there are :: in the end of the description, this change gets rid of them.